osquery/tables/system/windows/drivers.cpp

```diff
@@ -0,0 +1,199 @@
+/*
+*  Copyright (c) 2014-present, Facebook, Inc.
+*  All rights reserved.
+*
+*  This source code is licensed under the BSD-style license found in the
+*  LICENSE file in the root directory of this source tree. An additional grant
+*  of patent rights can be found in the PATENTS file in the same directory.
+*
+*/
+
+#define WIN32_LEAN_AND_MEAN
+#include <Windows.h>
+#include <Winsvc.h>
+#include <psapi.h>
+#include <string>
+
+#include <osquery/core.h>
+#include <osquery/logger.h>
+#include <osquery/tables.h>
+
+#include "osquery/tables/system/windows/registry.h"
+
+#pragma comment(lib, "Advapi32.lib")
+
+namespace osquery {
+namespace tables {
+
+const std::string kDrvStartType[] = {
+    "BOOT_START", "SYSTEM_START", "AUTO_START", "DEMAND_START", "DISABLED"};
+
+const std::string kDrvStatus[] = {"UNKNOWN",
+                                  "STOPPED",
+                                  "START_PENDING",
+                                  "STOP_PENDING",
+                                  "RUNNING",
+                                  "CONTINUE_PENDING",
+                                  "PAUSE_PENDING",
+                                  "PAUSED"};
+
+const std::map<int, std::string> kDriverType = {
+    {0x00000001, "KERNEL"}, {0x00000002, "FILE_SYSTEM"},
+};
+
+static std::map<std::string, std::string> loadedDrivers;
```

Comment:
As he's making use of this map throughout a couple of functions, is there any other option for populating the map aside from passing a `std::map<int, std::string> &`?  We had a discussion about this being a global and it seemed like @ExtremeGTX was hoping to avoid passing refs to multiple functions. Thoughts?